Tests.PhraseTMS/TestFiles/Input/Length explorer_en-US.html

```
@@ -0,0 +1,4 @@
+﻿<html lang="en-US"><head><meta name="blackbird-entry-id" content="3V1rm8jOm5OUtMLLiKWAUp"><meta name="blackbird-locale" content="en-US"><meta name="blackbird-ucid" content="3V1rm8jOm5OUtMLLiKWAUp"><meta name="blackbird-content-name" content="Length explorer"><meta name="blackbird-admin-url" content="https://app.contentful.com/spaces/12ktqqmw656e/environments/dev/entries/3V1rm8jOm5OUtMLLiKWAUp"><meta name="blackbird-system-name" content="Contentful"><meta name="blackbird-system-ref" content="https://www.contentful.com"></head><body its-rev-tool="Contentful" its-rev-tool-ref="https://www.contentful.com" its-rev-person="Vitalii Bezuhlyi"><div data-contentful-entry-id="3V1rm8jOm5OUtMLLiKWAUp"><h1 data-contentful-field-type="Symbol" data-contentful-field-id="title" data-blackbird-key="3V1rm8jOm5OUtMLLiKWAUp-title" data-blackbird-size="4,30">Length explorer</h1><div data-contentful-field-type="RichText" data-contentful-field-id="description" data-blackbird-key="3V1rm8jOm5OUtMLLiKWAUp-description"><p><p><strong>Improve your goldfish's physical fitness by getting him a bicycle.</strong></p>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)